dt-posts/dt-posts.php

```diff
@@ -681,6 +681,11 @@ public static function list_posts( string $post_type, array $search_and_filter_q
         if ( isset( $search_and_filter_query['dt_recent'] ) ){
             $data = self::get_recently_viewed_posts( $post_type );
         } else {
+
+            // Support wildcard searching between string tokens.
+            if ( !empty( $search_and_filter_query['text'] ) && !is_numeric( $search_and_filter_query['text'] ) ){
+                $search_and_filter_query['text'] = str_replace( ' ', '%', $search_and_filter_query['text'] );
+            }
```

Comment:
@kodinkat  Please move to https://github.com/DiscipleTools/disciple-tools-theme/blob/b37b4d22aa857a6042abd0cd0fe9c98ec54d83e5/dt-posts/posts.php#L1107